plugins/ci/cmd/insights-ci/main.go

```diff
@@ -217,7 +221,9 @@ func getTrivyReport(images []trivymodels.Image, configurationObject models.Confi
 		if !matchedImage && len(repoTags) > 0 {
 			imageRepo := repoTags[0]
 			imageRepo = strings.Split(imageRepo, ":")[0]
+			ID := fmt.Sprintf("%s@%s", imageRepo, sha)
```

Comment:
I think this needs to be `@sha256:`

Reply:
mmh aparently `config.Image` has this format `sha256:xxxxxx`
```
 "ArgsEscaped": true,
        "Image": "sha256:2bf39c9ea98d819cb507e965ebebdc87123b348d5127282c24782d43be52fc00",
        "Volumes": null,
        "WorkingDir": "",
```

Reply:
aha! good to know, thanks